com.tercad.fingrom.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.tercad.fingrom
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: fingrom
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --own-name=com.tercad.fingrom
+modules:
+  - name: Fingrom
+    buildsystem: simple
+    only-arches:
+      - x86_64
```

Comment:
this doesn't prevent building an aarch64 package.

Reply:
why not building from source? This would solve you other issue iwth GLIBC

Reply:
I see, Flutter....

Reply:
> this doesn't prevent building an aarch64 package.
As I see from https://github.com/flathub/im.fluffychat.Fluffychat/blob/master/im.fluffychat.Fluffychat.json it should work.

Reply:
You missed https://github.com/flathub/im.fluffychat.Fluffychat/blob/master/flathub.json#L2 and the docs https://docs.flathub.org/docs/for-app-authors/maintenance#dont-build-on-aarch64 ...

Reply:
How it differs from https://github.com/flathub/flathub/pull/4668/files#diff-449c727070caefacc6c03cb10922640e75b1ef6f2540973d535734f063aeb5f5R17  (lines 17-18)?

Reply:
You don't have a flathub.json that is necessary...

Reply:
Thanks, added